packages/language-server/src/completion/completions.ts

```diff
@@ -146,11 +164,6 @@ export function getSuggestionForFieldAttribute(
   wordsBeforePosition: string[],
   document: TextDocument,
 ): CompletionList | undefined {
-  // TODO type? suggestions for "@..." for type?
-  if (block.type !== 'model') {
-    return
-  }
-
```

Comment:
So before we did not do any completion in this case for MongoDB
```
type Something {
  text       String  |
}
```

And now it suggests `@db.` and the native types. This is untested and we should create an issue to add tests.

Reply:
Created https://github.com/prisma/language-tools/issues/1360